docs/oracle/economic-guarantees.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UMA's Optimistic Oracle is secured by Economic Guarantees
+sidebar_label: Economic guarantees
+---
+
+## UMA’s Approach to the Oracle Problem
```

Comment:
Generally think we should decide on a standard here, but I'm of the opinion that it looks weird to have a title after a title. IMO the first paragraph should always be an introduction.